### ISSUE TITLE
ES6 object-short-notation in directionEnabled not transformed to umd in dist

### ISSUE THREAD

**Issue by gafderks** [chartjs/chartjs-plugin-zoom#349]
`directionEnabled` below uses `{chart}`, which is ES6 object-short-notation.
https://github.com/chartjs/chartjs-plugin-zoom/blob/ac429049d8af5accfb2b049be57d0ebddd8c52a5/src/plugin.js#L87
The problem is that in the dist files created using Rollup, this syntax is preserved. Therefore it breaks tools such as gulp-uglify which output errors such as:
```
GulpUglifyError: unable to minify JavaScript
Caused by: SyntaxError: Unexpected token: punc «}»
```

**Comment by benmccann** [comment]
I sent a PR to fix https://github.com/chartjs/chartjs-plugin-zoom/pull/350

**Comment by benmccann** [comment]
I released a new version 0.7.7. It should be up shortly
